### PR TITLE
Add Samba instances

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds Samba instances.  These instances re-share the EFS file share via SMB so that they can be mounted by Windows instances.

## 💭 Motivation and context ##

Windows instances cannot mount the EFS share, and AWS FSx requires Active Directory, so this is the best solution for us.

## 🧪 Testing ##

I have tested these changed in env1 of our COOL staging environment.  I created two instances based on the [cisagov/samba-packer](https://github.com/cisagov/samba-packer) AMI and verified that each was sharing the EFS share via SMB.  I then mounted `Samba1`'s SMB share from `Samba0` using `mount -t cifs -o guest,uid=2048,gid=2048 //samba1/Share /mnt/smb` and verified that everything looked like it should.  I also created a text file in the EFS share and changed its ownership to `smbguest:efs_users` and verified that I could read and write it via the SMB share.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by jsf9k** [comment]
This PR seems to be stuck as a draft while the repo is private.  I think we found an edge case in GitHub's web UI!

**Review by dav3r — APPROVED** [review]
This all looks excellent to me- strong work as always!  💃

File: operations_acl_rules.tf [hunk 1]
```diff
@@ -128,8 +155,10 @@ resource "aws_network_acl_rule" "operations_ingress_from_anywhere_via_ports_1024
 # Allow ingress from anywhere via ephemeral TCP/UDP ports 3390-5900.
 #
 # For: Assessment team operational use, but we don't want to allow
-# public access to RDP on port 3389 or VNC on port 5901.
-resource "aws_network_acl_rule" "operations_ingress_from_anywhere_via_ports_3390_thru_5900" {
+# public access to RDP on port 3389 or Cobalt Strike Teamservers on port 50050.
+#
+# We can skip the VNC and WINRM ports as they are blocked by rules above.
```
Comment (mcdonnnj) [inline operations_acl_rules.tf]:
```suggestion
# We can skip the VNC and WinRM ports as they are blocked by rules above.
```

**Review by mcdonnnj — APPROVED** [review]
Strong work 💪💪💪
